x-pack/platform/packages/shared/kbn-inference-tracing/src/types.ts

```diff
@@ -47,11 +51,15 @@ export interface GenAISemConvAttributes {
   [GenAISemanticConventions.GenAIToolName]?: string;
   'input.value'?: any;
   'output.value'?: any;
-  [ElasticGenAIAttributes.InferenceSpanKind]?: 'CHAIN' | 'LLM' | 'TOOL';
+  [ElasticGenAIAttributes.InferenceSpanKind]?: 'CHAIN' | 'AGENT' | 'LLM' | 'TOOL';
   [ElasticGenAIAttributes.ToolDescription]?: string;
   [ElasticGenAIAttributes.ToolParameters]?: string;
   [ElasticGenAIAttributes.Tools]?: string;
   [ElasticGenAIAttributes.ToolChoice]?: string;
+  [ElasticGenAIAttributes.AgentId]?: string;
```

Comment:
Question: With adding these additional attributes, do we need to give broader visibility to these additions? Now with the GenAI Semver team disbanded, not sure who needs to be aware of this.